garp-on-svi-action-pack/garp-on-svi/script.py

```diff
@@ -0,0 +1,49 @@
+def string_to_list(string_to_convert):
+    numbers = []
+    segments = [ segment.strip() for segment in string_to_convert.split(",") ]
+    for segment in segments:
+        if "-" in segment:
+            for i in range(int(segment.split("-")[0]), int(segment.split("-")[1]) + 1):
+                numbers.append(i)
+        else:
+            numbers.append(int(segment))    
+    return numbers
+
+switch = ctx.getDevice()
+vlan_ids = ctx.changeControl.args['VLAN IDs']
+ctx.alog("{}".format(ctx.alog(vlan_ids)))
+
+if vlan_ids is not None:
+    vlan_ids = string_to_list(vlan_ids)
+else:
+    ctx.alog("No VLAN IDs provided")
+
+for vlan_id in vlan_ids:
+    ctx.alog("Getting interface info on {} for Vlan {}".format(switch.ip, vlan_id))
```

Comment:
These lines should not exceed 100 characters if possible, though there is a hard limit at 120.
Lines 23, 25, and 39 go past that hard limit, so at the very least they will need to be modified
Ideally we'd have everything here <100 characters long